.github/workflows/ci.yml

```diff
@@ -18,9 +19,6 @@ jobs:
           - version: '1.5'
             os: ubuntu-latest
             arch: x64
-          - version: 'nightly'
-            os: ubuntu-latest
-            arch: x64
```

Comment:
Please don't remove this.

Reply:
What are you thoughs on this?
https://discourse.julialang.org/t/why-do-packages-run-continuous-integration-tests-on-julia-nightly/101208/3?u=gdalle

Reply:
I'll undo the change for now but I think it makes sense

Reply:
I saw your issue on Graphs.jl, but was quite busy to react to it - lets discuss it there first and then decide on a action, is that alright for you?

Reply:
sounds good to me, I just wanted to update some docs and READMEs in the ecosystem, so the nightly testing is fairly independent